### PR TITLE
Move endorsements to the main page

### LINKED ISSUE #9: Put the main endorsements on the README.md page
What do you think about small refactoring and putting the names on the main page - README
I think it would look more inspiring like that.
What do you think?

### PR DESCRIPTION
**Pull request by suchov**
Some example to fix https://github.com/kottans/kottans/issues/9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,45 @@
 This repo contains documents regarding general
 [Kottans organization](https://github.com/kottans) matters.
 
-## Contents
+[Kottans badge](https://github.com/kottans/artifacts/blob/master/badges.md)
+
+# Endorsements
+
+This document is a register of
```
Comment (OleksiyRudenko) [inline README.md]:
```suggestion
This section contains a register of
```

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
Optional change though consistent.

File: README.md (hunk 1 above)
Comment (suchov) [reply README.md]: fixed

**Review by OleksiyRudenko — APPROVED** [review]
LGTM

**Pull request merged**